portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +26,7 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-		response.setContentType("text/html");
+	response.setContentType("text/html");
```

Comment:
I think there's still one extra space?

Reply:
Man this is so weird, I'm using Vim as my text editor and in Vim the indentation looks fine, but when I open it up in the actual cloud shell editor the indentation is off. 
![image](https://user-images.githubusercontent.com/37886623/83536617-76620480-a4c1-11ea-8e57-bc2982c334a6.png)
But then when I fix the indentation in my cloud editor, the indentation is off in vim!
![image](https://user-images.githubusercontent.com/37886623/83536708-94c80000-a4c1-11ea-8b24-8a57c5ca2dc0.png)
I'll try pushing it so it's correct in the cloud editor and hopefully that works? Not sure though haha.

Reply:
It might be off if tab sizing is represented differently in each editor. Perhaps you can disable tab spacing?

Reply:
Good point, let me give that a shot.

Reply:
Yep I think that was it!